packages/elements/tests/spec/link-with-icon_spec.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { render } from 'lit-html';
+import '../../src/components/link-with-icon/link-with-icon';
+import { Default } from '../../src/components/link-with-icon/link-with-icon-story';
+
+const template = (props?) =>
```

Comment:
Could we isolate tests per component folder? We've been following that pattern in the other packages, under a subfolder `__tests__`.